alpine/base-requirements.in

```diff
@@ -0,0 +1,35 @@
+# WSGI
+gunicorn
+gevent
+
+# Flask and related plugins
+flask
+flask-cors
+flask-apispec
+
+# Other common dependencies
+raven[flask]
+python-jose
+
+# Development (QA, testing, debugger tools)
+pytest
+pytest-cov
+black
+flake8
+flake8-docstrings
+flake8-bugbear
+isort
+safety
+ipython
+
+# Compatibility constraints
+
+# pydocstyle (dependency of flake8-docstrings): pin to v3.0.0 until
+# flake8-docstrings fixes compatibility with v4.0.0
```

Comment:
I'm not sure this is still an issue.

Reply:
I can try it out separately after all this is merged.